openapi/schemas/processes-core/statusCode.yaml

```
@@ -1,8 +1,13 @@
 type: string
 nullable: false
 enum:
-  - accepted
+  - created
+  - queued
   - running
+  - cancelled
```

```suggestion
  - canceled
```

openEO uses "canceled", is it double l in OGC API Processes?